doc/api/buffer.md

```diff
@@ -1264,7 +1264,7 @@ console.log(buf.lastIndexOf('buffer', 4));
 const utf16Buffer = Buffer.from('\u039a\u0391\u03a3\u03a3\u0395', 'ucs2');
 
 // Prints: 6
-console.log(utf16Buffer.lastIndexOf('\u03a3', null, 'ucs2'));
+console.log(utf16Buffer.lastIndexOf('\u03a3', buf.length, 'ucs2'));
```

Comment:
Maybe `null` could be changed into `undefined` for now (`undefined` does print `6` here). And if #9801 is fixed, we could restore the `null` .